### PR TITLE
Move Android and iOS folders.

### PR DESCRIPTION
**Pull request by DylanVann**
This is a breaking change.

I think this should be done to:

- Use a more standard React Native package directory structure.
- Reduce complexity (no `postinstall`).
- Reduce the probability of issues related to automatic linking and symlinking.

See: https://github.com/wix/react-native-interactable/issues/12#issuecomment-386686631

There are also several other open issues related to symlinks.

### CODE REVIEW

File: Interactable.podspec [hunk 1]
```diff
@@ -14,7 +14,7 @@ Pod::Spec.new do |s|
   s.platform     = :ios, "7.0"
 
   s.source       = { :git => "https://github.com/wix/react-native-interactable.git" }
-  s.source_files  = "lib/ios/Interactable/*.{h,m}"
+  s.source_files  = "ios/Interactable/*.{h,m}"
```
Comment (markhu) [inline Interactable.podspec]: As someone unfamiliar with `.podspec` I had to check at https://guides.cocoapods.org/syntax/podspec.html to assure myself that these curly braces weren't a problem here.

Comment (DylanVann) [reply Interactable.podspec]: Yeah, other project actually tend to use `ios/**/*.{h,m}`.

**Comment by jsispal** [comment]
@DylanVann - I'm still getting `Could not list contents of '/home/travis/build/jsispal/app/node_modules/react-native-interactable/{ios,android}'. Couldn't follow symbolic link.` on automated builds using this PR, any ideas? Running (Ubuntu 14.04.5 LTS).

**Comment by jsispal** [comment]
Thanks @DylanVann - this fixes the build issues (local and automated). Would be great to get this PR merged in 👍

**Comment by f0rr0** [comment]
can this be merged? @markhu @rjpizarro

**Comment by evanjmg** [comment]
Please merge this to fix symlinks

**Comment by traxx10** [comment]
Am also getting the same issue

**Comment by krszwsk** [comment]
We're also experiencing this issue:

```
Could not list contents of '...react-native-interactable...'. Couldn't follow symbolic link.
```

**Comment by rotemmiz** [comment]
I am closing the PR as a part of an effort to cleanup and revive the project. If this issue persists after v1.0.0, you are more than welcome to reintroduce the PR.

Thanks for your contribution.

**Comment by DylanVann** [comment]
I don't think closing PRs that fix persistent issues and make setup easier for people is a good way to go about "reviving the project".

I won't be reintroducing the PR.

**Comment by rotemmiz** [comment]
No need to reintroduce the PR, this is already solved in v1.0.0-alpha.1
https://www.npmjs.com/package/react-native-interactable/v/1.0.0-alpha.1

V1 is coming soon

**Comment by DylanVann** [comment]
Yeah I saw that, although you didn't mention it here. I understand closing PRs if you have merged another superior fix, but just closing PRs to "cleanup the project" strikes me as inconsiderate.

You may want to also check that `react-native link` also works well with whatever fix you implemented.

**Comment by DylanVann** [comment]
Trust me I can understand wanting to close all open issues and PRs on a project. I would resist the urge though, because people put time and effort into reporting issues and trying to make contributions.

I'm definitely guilty of ignoring PRs and issues, but I'd favor that over closing them without giving a reason. Anyways, no hard feelings, open source is hard 🤷‍♂️

**Pull request closed without merging**